src/packagedcode/podspec.py

```diff
@@ -0,0 +1,339 @@
+            stripped_authors = stripped_authors.strip()
+            stripped_authors = stripped_authors.split(',')
+            author_names = []
+            author_email = []
+            for split_author in stripped_authors:
+                split_author = split_author.strip()
+                author, email = parse_person(split_author)
+                author_names.append(author)
+                author_email.append(email)
+            podspec_data['author'] = author_names
+            podspec_data['email'] = author_email
+        elif '.summary' in line:
+            summary = re.sub(r"/*.*summary.*?=", "", line)
+            stripped_summary = summary.replace("'", "")
+            stripped_summary = stripped_summary.replace('"', "")
+            stripped_summary = stripped_summary.strip()
+            podspec_data['summary'] = stripped_summary
+        elif '.description' in line:
+            podspec_data['description'] = get_description(location)
+        elif '.homepage' in line:
+            homepage_url = re.sub(r"/*.*homepage.*?=", "", line)
+            stripped_homepage_url = homepage_url.replace("'", "")
+            stripped_homepage_url = stripped_homepage_url.replace('"', "")
+            stripped_homepage_url = stripped_homepage_url.strip()
+            podspec_data['homepage_url'] = stripped_homepage_url
+        elif '.dependency' in line:
+            match = add_dependency_regex(line)
+            line = match.group('line')
+            podspec_data['dependencies'] = parse_line(line)
+
+    deps = OrderedDict()
+    dependency = podspec_data.get('dependencies')
+    if dependency:
+        for dep in dependency:
+            dep_name = dep[0]
+            dep.pop(0)
+            dep_version = dep
+            deps[dep_name] = dep_version
+    podspec_data['dependencies'] = deps
+    return podspec_data
+
+
+def get_description(location):
+    """
+    https://guides.cocoapods.org/syntax/podspec.html#description
+    description is in the form:
+    spec.description = <<-DESC
+                     Computes the meaning of life.
+                     Features:
+                     1. Is self aware
+                     ...
+                     42. Likes candies.
+                    DESC
+
+    This function will return the multiline desciption.
+    """
+    contents = io.open(location, encoding='utf-8', closefd=True).readlines()
+    description = ''
+    for i, content in enumerate(contents):
+        if '.description' in content:
+            for cont in contents[i+1:]:
+                if 'DESC' in cont:
+                    break
+                description += ' '.join([description, cont.strip()])
+            break
+    return description
+
+def parse_person(person):
+    """
+    https://guides.cocoapods.org/syntax/podspec.html#authors
+    Author can be in the form:
+        s.author = 'Rohit Potter'
+        or
+        s.author = 'Rohit Potter=>rohit@gmail.com'
+
+    Author check:
```

Comment:
When you use doctests, introduce them not with: `Author check` but rather: `For example:`